packages/salesforcedx-vscode-lwc/src/messages/i18n.ts

```diff
@@ -31,10 +31,10 @@ export const messages = {
     'The local development server exited unexpectedly with code %s.',
   force_lightning_lwc_start_already_running:
     'The local development server is already running.',
-  force_lightning_lwc_stop_text: 'SFDX: Stop Local Development Server',
+  force_lightning_lwc_stop_text: 'SFDX: Stop LWC Code Preview Server',
   force_lightning_lwc_stop_not_running:
     'The local development server is not running.',
-  force_lightning_lwc_stop_in_progress: 'Stopping local development server',
+  force_lightning_lwc_stop_in_progress: 'Stopping local development Server',
```

Comment:
`Stopping LWC Code Preview Server` ?

Reply:
To be coherent it should be something like  'Stopping LWC Code Preview Server', right?

Reply:
Done!